pkg/archive/jar_test.go

```diff
@@ -180,7 +180,7 @@ func TestStableJARBuildMetadata(t *testing.T) {
 	}
 }
 
-func TestStableOrderOfAttributeValues(t *testing.T) {
+func TestStableJAROrderOfAttributeValues(t *testing.T) {
```

Comment:
@msuozzo note that I also changed the name of the test to follow naming conventions.

Reply:
yep sounds good!